teams/MAINTAINERS

```diff
@@ -3,7 +3,7 @@ DrSOKane
 martinjrobins
 rtimms
 Saransh-cpp
-tinosulzer
+valentinsulzer
```

Comment:
We also need to update this in [https://github.com/pybamm-team/PyBaMM/blob/develop/.all-contributorsrc](https://github.com/pybamm-team/PyBaMM/blob/develop/.all-contributorsrc) or we can leave it as it is to avoid it getting moved to contributors group instead while regenerating teams

Reply:
I think we should update this there as well, thanks for catching this, @arjxn-py. Could you write a PR? Once we merge it in the PyBaMM repo, we can merge this.